### PR TITLE
fedora rpm spec file update

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: opae.spec [hunk 1]
```diff
@@ -138,68 +129,68 @@ DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolmmlink -P cmake_install.cmake
...
 
 %post
-ldconfig
-
-%postun
-ldconfig
-
-%pre
+mkdir -p %{_sysconfdir}/ld.so.conf.d
+echo "" > %{_sysconfdir}/ld.so.conf.d/opae-c.conf
```
Comment (tswhison) [inline opae.spec]: Why is this needed?

File: opae.spec [hunk 2]
```diff
@@ -138,68 +129,68 @@ DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolmmlink -P cmake_install.cmake
...
-
-%files samples
-%defattr(-,root,root,-)
 %{_bindir}/hello_fpga
-%{_bindir}/hello_events*
 %{_bindir}/object_api
+%{_bindir}/hello_events
 %{_bindir}/hello_cxxcore
+%{_bindir}/afu_json_mgr
+%{_bindir}/packager
```
Comment (tswhison) [inline opae.spec]: we are upstreaming packager?

Comment (anandaravuri) [reply opae.spec]: including in 1.4.1 release, converting rbf file to gbs.

File: opae.spec (hunk 1 above)
Comment (anandaravuri) [reply opae.spec]: fixed
